Makefile

```diff
@@ -28,7 +28,7 @@ precommit: lint test
 
 test:
 	# We use one thread so the emscripten stdouts doesn't collide
-	cargo test -- --test-threads=1 $(runargs) 
+	cargo test test_nginx # -- --test-threads=1 $(runargs)
```

Comment:
I think you committed this one by mistake.

Reply:
I went ahead and switched it to a shell script. This should be reverted now :)